### PR TITLE
ADDED: School Admission Form-update

### PR DESCRIPTION
**Pull request by surendran18022002**

### CODE REVIEW

File: Surendran/School Admission Form/output.html [hunk 1]
```diff
@@ -42,103 +49,37 @@
 	<script>
      var op=localStorage.getItem("txtvalue");
```
Comment (NavinAT) [inline Surendran/School Admission Form/output.html:50]: Indent the code properly

File: Surendran/School Admission Form/style/style.css [hunk 2]
```diff
@@ -59,8 +56,8 @@ margin: 100px;
 
 	width: 100%;
  font-size: 80px;
-background: linear-gradient(
-            to right ,#b78628, #c69320, #dba514, #eeb609);
+background: radial-gradient(
```
Comment (NavinAT) [inline Surendran/School Admission Form/style/style.css:59]: Indent the CSS code properly.

File: Surendran/School Admission Form/.vscode/launch.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (NavinAT) [inline Surendran/School Admission Form/.vscode/launch.json:1]: Remove this .vscode folder

File: Surendran/School Admission Form/edit.html [hunk 4]
```diff
@@ -3,114 +3,134 @@
...
 	<meta charset="utf-8">
 	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<link rel="stylesheet" type="text/css" href="style.css">
-	<link rel="html" type="text/html" href="output.html">
-	<link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+	<link rel="stylesheet" type="text/css" href="style/style.css">
+	
 <link href="https://fonts.googleapis.com/css2?family=Cookie&family=Dancing+Script&family=Finger+Paint&family=Rampart+One&family=Sassy+Frass&display=swap" rel="stylesheet">
+<script type="text/javascript" src="js/intex.js"></script>
```
Comment (NavinAT) [inline Surendran/School Admission Form/edit.html:10]: Indent the Html file properly

File: Surendran/School Admission Form/js/intex.js [hunk 5]
```diff
@@ -0,0 +1,128 @@
+
+    
+                                         
+                   
+               
+ function save() {
```
Comment (NavinAT) [inline Surendran/School Admission Form/js/intex.js:6]: Indent the code properly.

File: Surendran/School Admission Form/output.html [hunk 6]
```diff
@@ -42,103 +49,37 @@
...
      var obj=JSON.parse(op);
+
```
Comment (NavinAT) [inline Surendran/School Admission Form/output.html:52]: Move the Script to separate JS file

**Review by NavinAT — COMMENTED** [review]
Please see the review comments and do the changes.

File: Surendran/School Admission Form/output.html (hunk 1 above)
Comment (surendran18022002) [reply Surendran/School Admission Form/output.html:50]: completed

File: Surendran/School Admission Form/style/style.css (hunk 2 above)
Comment (surendran18022002) [reply Surendran/School Admission Form/style/style.css:59]: completed

File: Surendran/School Admission Form/.vscode/launch.json (hunk 3 above)
Comment (surendran18022002) [reply Surendran/School Admission Form/.vscode/launch.json:1]: completed

File: Surendran/School Admission Form/edit.html (hunk 4 above)
Comment (surendran18022002) [reply Surendran/School Admission Form/edit.html:10]: completed

File: Surendran/School Admission Form/js/intex.js (hunk 5 above)
Comment (surendran18022002) [reply Surendran/School Admission Form/js/intex.js:6]: completed

**Pull request merged**